### PR TITLE
Layout change and added progress bar: Removed Section From layout and Added Nprogress to show page chnage, kept Main wrapper as it is.

### PR DESCRIPTION
**Pull request by chiragpadyal**
Removed Section From layout and Added Nprogress to show page chnage, kept Main wrapper as it is.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
+import type { NextPage } from 'next';
 import IntroPage from '../components/intropage/intropage';
 import Overview from '../components/Overview/overview';
 import Footer from '../components/Footer/footer';
 import Main from '../layout/Main';
 
-function Home() {
+const Home: NextPage = () => {
```
Comment (YadavGulshan) [inline pages/index.tsx:7]: can you explain me the benefit of this change?

Comment (chiragpadyal) [reply pages/index.tsx:7]: nothing just used arrow function as it is short

Comment (chiragpadyal) [reply pages/index.tsx:7]: its native index page style of next js

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,8 +1,28 @@
-import '../styles/globals.css';
+import * as React from 'react';
```
Comment (YadavGulshan) [inline pages/_app.tsx:1]: Try importing the only thing you want... Don't import everything.

you may check [this](https://stackoverflow.com/questions/54585763/what-is-the-difference-between-import-as-react-from-react-vs-import-react-fr)

File: yarn.lock [hunk 3]
```diff
@@ -2027,6 +2032,11 @@ npm-run-path@^4.0.1:
   dependencies:
     path-key "^3.0.0"
 
+nprogress@^0.2.0:
```
Comment (YadavGulshan) [inline yarn.lock:2035]: can you explain the need of this package in our project?

**Review by YadavGulshan — COMMENTED** [review]
Hey, I have some queries with your PR. That would be great if you answer them  :smile:

**Pull request closed without merging**